### DISCUSSION TITLE
Strategy Roasters

### DISCUSSION THREAD

**Discussion by Heiwa1580** [Glimesh/company#28] [Business Partnerships]
Hi All,

I wanted to bring this up. It goes back a bit but I would love to explore this

https://twitter.com/StrategyCoffee/status/1379545381986504705?s=20

Strategy Roasters is owned by https://twitter.com/NickIsTheWay

I think that this is sort of a win win for us both. We get some new little hype we can bring around the Glimesh Coffee line with them and something for the Community to grab and Strategy Coffee gets some extra publicity and sales. 

I don't really see any negatives to the idea. Anyone else have thoughts?

**Comment by Heiwa1580** [comment]
Benefits to Us

- A "custom" coffee line that we can share with our community
- Co advertising in the form of being shared by Strategy
- Some minor earnings of shared profits on coffee

Benefits to Strategy

- Our community buying the coffee is sales to them
- Co advertising in us sharing that we have a coffee line there
- The potential of consistent sales if people like the Glimesh blend

**Comment by Heiwa1580** [comment]
Was there any other thoughts? I'd like to reach out to see about the potential of making this happen with them

**Reply by clone1018** [reply]
I'm not bought into the idea that a coffee line is right for our business right now. The benefits don't seem to be anything immediately valuable and I personally don't like the idea that they will "co-advertise" with us only after we start selling their coffee. 

I think we should focus on partnerships that bring us direct traffic to our main product, a streaming site, as opposed to an unrelated merch line.

**Reply by Heiwa1580** [reply]
Sounds good. I agree with that and I think we can close the discussion from there. Thanks Clone
